torch/_export/constraints.py

```diff
@@ -13,7 +30,7 @@ def constrain_as_value(symbol, min: Optional[int] = None, max: Optional[int] = N
     Add min/max constraint on the intermediate symbol at tracing time
     """
 
-    constrain_range(symbol, min=min, max=max)
+    _constrain_range(symbol, min=min, max=max)
```

Comment:
Why not just inline `_constrain_range`?

Reply:
Will do. Mostly some outcome of changes between revisions but yeah it makes sense to make it inline.